elliottlib/cli/advisory_commons_cli.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import, print_function, unicode_literals
+from elliottlib import logutil
+from elliottlib.cli.common import cli
+from elliottlib.util import green_prefix
+from errata_tool import Erratum, ErrataException
+import click
+
+LOGGER = logutil.getLogger(__name__)
+supported_update_fields = [
+    'publish_date',
+    'text_only',
+    'text_only_cpe',
+    'owner_email',
+    'manager_email',
+    'manager_id',
+    'qe_email',
+    'qe_group',
+]
+
+
+@cli.command("advisory-commons", short_help="Show or update advisory commons for a group of advisories")
+@click.option("--advisories", metavar='ADVISORY',
+              help="Select given advisory id(s). comma separated")
+@click.option("--field", required=True,
+              help="Advisory field (like publish_date) to show or update. Examples of show only fields: ["
+                   "'errata_state', "
+                   "'errata_name', 'content_types', 'synopsis', "
+                   "'errata_bugs', "
+                   "'errata_type']")
+@click.option("--new",
+              help=f"New value of field to update advisories with. --field should be one of"
+                   f" {supported_update_fields}")
+@click.option("--noop", "--dry-run",
+              is_flag=True,
+              default=False,
+              help="Do not actually change anything")
+@click.pass_obj
+def advisory_commons_cli(runtime, advisories, field, new, noop):
+    """Display or Change a common field (like date) across multiple advisories.
+
+Advisories created for an OCP version have common fields, that sometimes
+will need updating. This command helps with that.
+
+    NOTE: The two advisory input options (--assembly and --advisories)
+    are mutually exclusive and can not be used together.
+
+    Show the field "publish_date" for all advisories for an assembly
+
+    $ elliott -g openshift-4.8 --assembly 4.8.8 advisory-commons --field "publish_date"
+
+    Change the field "publish_date" for all advisories for an assembly
+
+    $ elliott -g openshift-4.8 --assembly 4.8.8 advisory-commons --field "publish_date" --new "2021-Aug-31"
+
+    Do not actually change anything
+
+    $ elliott -g openshift-4.8 --assembly 4.8.8 advisory-commons --field "publish_date" --new "2021-Aug-31" --noop
+"""
+    count_flags = sum(map(bool, [runtime.assembly != 'stream', advisories]))
+    if count_flags > 1:
+        raise click.BadParameter("Use only one of --assembly or --advisories")
```

Comment:
For releases without assembly support (like 3.11), `runtime.assembly` is `None`.

BTW I don't think this check is absolutely needed.

Reply:
fixed!